edge/conf/edge.yaml

```diff
@@ -44,3 +45,9 @@ edged:
     maximum-dead-containers-per-container: 1
     docker-address: unix:///var/run/docker.sock
     version: 2.0.0
+    runtime-type: docker
+    remote-runtime-endpoint: /var/run/containerd/containerd.sock
+    remote-image-endpoint: /var/run/containerd/containerd.sock
+    runtime-request-timeout: 2
+    podsandbox-image: k8s.gcr.io/pause
+    kubelet-root-dir: /home/root1/
```

Comment:
should this be renamed to "edged-root-dir" ? is this the dir where edge_core binary is present ? 
can we have this like  a standard directory location like "/etc/kubeedge" ?

Reply:
Changed as specified